tests/unit/store/traction/pacbio/runCreate/actions.spec.js

```diff
@@ -377,4 +334,74 @@ describe('actions.js', () => {
       expect(errors).toEqual(['it didnt work'])
     })
   })
+
+  describe('setInstrumentData', () => {
+    it('when a key is passed', () => {
+      const commit = vi.fn()
+      const instrumentTypeKey = 'SequelIIe'
```

Comment:
Instead of hardcoding `'SequelIIe'` or `'Revio'`, could it  be pulled in from the config?